pimsviewer/viewer.py

```diff
@@ -729,17 +747,35 @@ def to_clipboard(self):
     def _get_all_files_in_dir(directory, extensions=None):
         if extensions is None:
             file_list = [f for f in listdir(directory) if isfile(join(directory, f))]
-
         else:
             file_list = [f for f in listdir(directory) if isfile(join(directory, f))
                          and drop_dot(os.path.splitext(f)[1]) in extensions]
 
         return sorted(file_list, key=natural_keys)
 
+    def update_color_mode(self, color=None):
+        """Updates the color mode (full color/greyscale) of the current image.
+        """
+        self.force_color = self._available_colors[color]
+        self.update_view()
 
-    # def to_frame(self):
-    #     return Frame(rgb_view(self.to_pixmap().toImage()),
-    #                  frame_no=self.index['t'])
+    def _generate_color_menu(self):
```

Comment:
Just for code clarity and to prevent the `__init__` from becoming very long.
But I moved it to the init function.